pipeline/transformer/transformer_marc_common/src/main/scala/weco/pipeline/transformer/marc_common/transformers/MarcNotes.scala

```diff
@@ -0,0 +1,132 @@
+package weco.pipeline.transformer.marc_common.transformers
+import grizzled.slf4j.Logging
+import weco.catalogue.internal_model.work.{Note, NoteType}
+import weco.pipeline.transformer.marc_common.models.{
+  MarcField,
+  MarcRecord,
+  MarcSubfield
+}
+import weco.pipeline.transformer.text.TextNormalisation._
+
+import java.net.URL
+import scala.util.Try
+trait MarcNotes extends Logging {
+
+  protected val globallySuppressedSubfields: Set[String] = Set("5")
+
+  val notesFields: Map[String, MarcField => Note] = Map(
```

Comment:
This list is yanked from SierraNotes.  The significant differences being that I've moved the privacy condition into createOwnershipNote, and removed (moved) the Sierra-specific entries.